Funcky.Xunit/FunctionalAssert/Error.cs

```diff
@@ -1,31 +1,32 @@
 using System.Diagnostics.CodeAnalysis;
 using Xunit.Sdk;
+using static Xunit.Sdk.ArgumentFormatter;
 
 namespace Funcky;
 
 public static partial class FunctionalAssert
 {
     /// <summary>Asserts that the given <paramref name="result"/> is <c>Error</c>.</summary>
-    /// <exception cref="AssertActualExpectedException">Thrown when <paramref name="result"/> is <c>Ok</c>.</exception>
+    /// <exception cref="XunitException">Thrown when <paramref name="result"/> is <c>Ok</c>.</exception>
```

Comment:
This is technically a breaking change because we documented the exception to be `AssertActualExpectedException` before, but I don't think anyone depends on the exact exception here...